src/platforms/native/platform.c

```diff
@@ -485,3 +487,17 @@ static void setup_vbus_irq(void)
 
 	exti15_10_isr();
 }
+
+static void wait_for_usb_host_config_request(void)
+{
+	// Toggle idle/running LED while waiting for USB serial or
+	// DFU set configuration requests
+	while (usb_get_config() == 0)
+	{
+		// TODO use platform_timeout and poll for expiry to toggle LED
+		platform_delay(300);
+		SET_ERROR_STATE(true);
+		platform_delay(300);
+		SET_ERROR_STATE(false);
+	}
+}
```

Comment:
Please add a new line at the end of the file, and run clang-format to format the loop to the project's code style.

Reply:
Got it, thanks